packages/core/less/message-dialog.less

```diff
@@ -0,0 +1,26 @@
+.message-dialog-pf-body {
+  display: flex;
+  color: #363636;
+}
+
+.message-dialog-pf-icon {
+  font-size: 30px;
+  margin-right: 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.message-dialog-pf-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.message-dialog-pf-text-primary {
+  font-size: 16px;
+  margin-bottom: 10px;
```

Comment:
We already have a `lead` class from bootstrap which has similar styles based on variables. 

Can we remove these styles and use something like this instead:
```
<div class="message-dialog-pf-text-primary">
  <p class="lead">Primary Text</p>
</div>
```

Reply:
Actually, can we get rid of the primary and secondary divs altogether, and just have this?:
```
<div class="message-dialog-pf-text">
  <p class="lead">Primary Text</p>
  <p>You are about to do something</p>
  <ul>...</ul>
</div>
```